lib/weather/presentation/weather_details.dart

```diff
@@ -1,10 +1,361 @@
 import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:yet_another_weather_app/widgets/app_icons_icons.dart';
+
+import '../../widgets/temperature_data.dart';
+
+const Color blackColor = Color.fromRGBO(51, 40, 33, 1);
 
 class WeatherDetails extends StatelessWidget {
   const WeatherDetails({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return const Placeholder();
+    return Scaffold(
+      extendBodyBehindAppBar: true,
+      appBar: AppBar(
+        elevation: 0,
+        shadowColor: Colors.transparent,
+        backgroundColor: Colors.transparent,
+        foregroundColor: blackColor,
+        leading: IconButton(
+          icon: const Icon(
+            Icons.arrow_back,
+          ),
+          onPressed: () {
+            Navigator.pop(context);
+          },
+        ),
+      ),
+      body: SingleChildScrollView(
+        padding: const EdgeInsets.fromLTRB(16, 100, 16, 16),
+        child: Stack(
+          children: [
+            Transform.translate(
+              offset: const Offset(80, -140),
+              child: Image.asset(
+                'assets/images/sun_cloud_angled_rain.png',
+                alignment: Alignment.topRight,
+              ),
+            ),
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  'Berlin,\nGermany',
+                  style: GoogleFonts.alegreyaSans(
```

Comment:
In "normal" project we should have styles predefined to not repeat ourself everywhere. Here's fine.
We will improve it in this task https://jira.viessmann.com/browse/FLUTE-15